### DISCUSSION TITLE
tags  created in gitlens  do not show  on github

### DISCUSSION THREAD

**Discussion by collisiondetection** [gitkraken/vscode-gitlens#1577] [Q&A] [answered]
Are tags meant to be only local to vscode as I can't see them showing on GitHub ?

**Comment by deadnews** [comment, marked as answer]
To push the tag to the remote server, you can open the Command Palette (`Ctrl` + `Shift` + `P`) and choose `Git: Push Tags` or `Git: Push (Follow Tags)`.

Also settings:
- vscode: `git.followTagsWhenSync` - Follow tags when running Git: Sync.
- git: `git config --global push.followTags true`
